english/Truth of the life of this world

```diff
@@ -0,0 +1,41 @@
+All living things and human beings have predetermined purposes to fulfil in life. These purposes owe their existence to the fact that everything has been created. An intelligent person notices that plan, design and wisdom exist in every detail of the infinitely varied world. This draws him to recognition of the Creator. He further concludes that since all living things are not the consequences of a random or mindless process they all serve important purposes. In the Qur'an, the last surviving authentic revealed guide to the true path for humanity, Allah repeatedly reminds us of the purpose of our life, which we tend to forget, and thereby summons us to clarity of mind and consciousness.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.